controllers/ovn_common.go

```diff
@@ -31,13 +31,15 @@ import (
 const (
 	tlsField                = ".spec.tls.secretName"
 	caBundleSecretNameField = ".spec.tls.caBundleSecretName" // #nosec
+	metricsTLSField         = ".spec.metricsTLS.secretName"
```

Comment:
can we expect tls.caBundleSecretName and metricsTLS.caBundleSecretName to be always the same? if so I think its ok to not watch it, otherwise we probably should.